src/main/scala/com/lightbend/kafka/scala/streams/KStreamS.scala

```diff
@@ -110,18 +106,31 @@ class KStreamS[K, V](val inner: KStream[K, V]) {
     inner.process(processorSupplierJ, stateStoreNames: _*)
   }
 
-  def groupByKey(): KGroupedStreamS[K, V] =
-    inner.groupByKey()
-
-  def groupByKey(serialized: Serialized[K, V]): KGroupedStreamS[K, V] =
-    inner.groupByKey(serialized)
-
-  def groupBy[KR](selector: (K, V) => KR): KGroupedStreamS[KR, V] = {
-    inner.groupBy(selector.asKeyValueMapper)
-  }
-
-  def groupBy[KR](selector: (K, V) => KR, serialized: Serialized[KR, V]): KGroupedStreamS[KR, V] = {
-    inner.groupBy(selector.asKeyValueMapper, serialized)
+  /**
```

Comment:
I'm not sure if I saw this, but do the contests explicitly confirm all three usage patterns. Specifically, I'm not sure I saw that #1 was being tested.

Reply:
You are correct! I added that test ..

Reply:
Great!